src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/RemoveLiquidityDialogV1.tsx

```diff
@@ -242,6 +243,7 @@ export const RemoveLiquidityDialogV1: React.FC<IRemoveLiquidityDialogV1Props> =
                 tx.loading || !valid || !canInteract || removeliquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-withdrawal-button"
```

Comment:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```

Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals